src/models/user.js

```diff
@@ -3,6 +3,10 @@ var _ = require("lodash");
 module.exports = User;
 
 function User(attr) {
+	// TODO: Remove this
+	attr.name = attr.name || attr.nick;
+	attr.mode = attr.mode || (attr.modes && attr.modes[0]) || "";
```

Comment:
One of these "will be nice to do at some point" things, not really important or blocking.